templates/static/js/src/demobox/components/LiveDemoBox/LiveDemoBox.tsx

```diff
@@ -18,44 +18,121 @@ type InitialState = {
+          });
+        };
+        setDefaultDropdownValues("type");
+        setDefaultDropdownValues("theme");
+        setDefaultDropdownValues("style");
+
+        const setDefaultSwitchValues = () => {
+          json.switch.map((switchOption: any) => {
+            setSelectValues({ ...selectValue, [switchOption.query]: false });
+          });
+        };
+        setDefaultSwitchValues();
       } catch (error) {
-        console.log('error', error);
+        console.log("error", error);
       }
     };
     fetchData();
   }, []);
 
+  const handleChange = (
+    e: React.ChangeEvent<HTMLSelectElement | HTMLInputElement>
+  ) => {
+    if (e.target.value === "on") {
+      setSelectValues({
+        ...selectValue,
+        [e.target.name]: !selectValue[e.target.name],
+      });
+    } else {
+      setSelectValues({
+        ...selectValue,
+        [e.target.name]: e.target.value.toLowerCase(),
+      });
+    }
+  };
+
+  const url = `/docs/examples/patterns/notifications/toast?type=${selectValue.type}&style=${selectValue.style}&actions=${selectValue.actions}&dismiss=${selectValue.dismiss}&timestamp=${selectValue.timestamp}`;
+
   return (
     <section className="p-strip--light">
       {configValues && configValues.dropdown && configValues.switch && (
         <form>
-          <div className="row" style={{gridGap: 0}}>
+          <div className="row" style={{ gridGap: 0 }}>
             <div className="col-2">
-              <Select id="select-type" label="select-type" name="Type" options={configValues.dropdown.type} />
+              <Select
+                id="type"
+                label="Type"
+                name="type"
+                onChange={handleChange}
+                options={configValues.dropdown.type}
+              />
             </div>
             <div className="col-2">
-              <Select id="select-theme" label="select-theme" name="Theme" options={configValues.dropdown.theme} disabled />
+              <Select
+                id="theme"
+                label="Theme"
+                name="theme"
+                onChange={handleChange}
+                options={configValues.dropdown.theme}
+                disabled
+              />
             </div>
             <div className="col-2">
-              <Select id="select-style" label="select-style" name="Style" options={configValues.dropdown.style} />
+              <Select
+                id="style"
+                label="Style"
+                name="style"
+                onChange={handleChange}
+                options={configValues.dropdown.style}
+              />
             </div>
           </div>
-          <div className="row" style={{gridGap: 0}}>
+          <div className="row" style={{ gridGap: 0 }}>
             <div className="p-card col-6">
               <h3>Example</h3>
-              <p className="p-card__content">space for exmaple here</p>
+              <p className="p-card__content">
+                <div>
```

Comment:
I see an error in the console that says `<div> cannot appear as a descendant of <p>`. I think we can just change the `p` to `div`.